python/shark_turbine/dynamo/backends/cpu.py

```diff
@@ -73,7 +73,7 @@ def _base_backend(gm: torch.fx.GraphModule, example_inputs):
     importer.import_graph_module(gm)
     print(module, file=sys.stderr)
     with context:
-        pm = PassManager.parse("builtin.module(torch-to-iree)")
+        pm = PassManager.parse("builtin.module(torch-to-iree,iree-util-import-resources)")
```

Comment:
I don't think you should need this? (probably just was for testing?)

Reply:
Yeah, this is not needed. Thanks for spotting that.